fe1-web/src/features/rollCall/components/EventRollCall.tsx

```diff
@@ -57,7 +57,7 @@ const EventRollCall = (props: IPropTypes) => {
         );
         return;
       }
-      requestOpenRollCall(event.idAlias).catch((e) =>
+      requestReopenRollCall(event.idAlias).catch((e) =>
         console.debug('Unable to send Roll call re-open request', e),
       );
```

Comment:
Is there a specific reason why in the case of reopening a roll call you have to press an additional button to get to the QR code scanning screen ("Scan Attendees" with onClick=`onScanAttendees`) rather than directly opening this screen as it is the case when opening the roll call for the first time (Line 67)?

Reply:
No specific reason for now. Thinking about it, it would make more sense with the current code to make it the same as the opening behavior.
Yes I noticed the bad coverage thing, might convert the pull request to draft while I take care of those...